content/en/blog/2022-08-26-manage-kyverno-policies-as-ocirepositories/index.md

```diff
@@ -0,0 +1,208 @@
+
+The Flux CLI generates a single layer OCI image for storing things. As you can use some other tools to generate an OCI image with multiple-layers in it, you can use [Layer Selection](https://github.com/fluxcd/flux2/tree/main/rfcs/0003-kubernetes-oci#layer-selection) feature that Flux provides to select the layers you want to use in the OCI image. If the layer selector matches more than one layer, the first layer matching the specified media type will be used. Note that Flux requires that the OCI layer is compressed in the tar+gzip format.
+
+{{< imgproc twitter-ss Resize 400x >}}
+{{< /imgproc >}}
+
+{{< imgproc meme Resize 400x >}}
+{{< /imgproc >}}
+
+Today, we’ll leverage the OCI Repositories feature to apply Kyverno policies stored in an OCI registry into the Kubernetes cluster.
+
+First, we need to install Flux CLI, please see the [installation](https://fluxcd.io/docs/installation/) page to find out details.
+
+Next, we should have Kubernetes cluster running, we’ll be using [KinD](https://kind.sigs.k8s.io/docs/user/quick-start#configuring-your-kind-cluster) for this purpose.
+
+```shell
+$ kind create cluster
+```
+
+Once the cluster has been provisioned successfully, we need to install Flux components into it by simply running the command below:
+
+```shell
+$ flux bootstrap github \
+  --owner=developer-guy \
+  --repository=flux-kyverno-policies \
+  --path=clusters/local \
+  --personal
+```
+
+> ⚠️ **Note:** Don't forget to change the values with your own details!
+
+This command will install Flux and create necessary files for us and push them into the repository.
+
+Next, we should install Kyverno first. As we do everything GitOps in mind, we’ll also install Kyverno with Flux. In order to do that, we use HelmRepository and HelmRelease resources:
+
+```yaml
+---
+apiVersion: source.toolkit.fluxcd.io/v1beta2
+kind: HelmRepository
+metadata:
+  name: kyverno
+  namespace: flux-system
+spec:
+  interval: 10m
+  url: https://kyverno.github.io/kyverno/
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: kyverno
+  namespace: flux-system
+spec:
+  interval: 10m
+  targetNamespace: kyverno
+  install:
+    createNamespace: true
+  chart:
+    spec:
+      chart: kyverno
+      sourceRef:
+        kind: HelmRepository
+        name: kyverno
+```
+
+Do not forget to check whether everything works fine before moving into the next steps:
+
+```shell
+$ kubectl get pods -n kyverno
```

Comment:
Please replace this with `flux get hr kyverno`, if there is anything wrong with the install, Flux will report it on the HelmRelease status.

Reply:
amazing, thx